kotlin/nearest-vertex/src/main/java/com/esri/arcgisruntime/sample/nearestvertex/MainActivity.kt

```diff
@@ -63,6 +61,9 @@ class MainActivity : AppCompatActivity() {
     activityMainBinding.coordinateDistanceTextView
   }
 
+  // California zone 5 (ftUS) state plane coordinate system.
+  private val statePlaneCaliforniaZone5 = SpatialReference.create(2229)
```

Comment:
```suggestion
  private val statePlaneCaliforniaZone5SpatialReference = SpatialReference.create(2229)
```

Just think it'll be easier to read later that the PointCollection(...) is taking a spatial reference as an argument